src/segment-loader.js

```diff
@@ -667,18 +672,15 @@ export default class SegmentLoader extends videojs.EventTarget {
                                                    this.sourceUpdater_.buffered());
 
     // Update segment meta-data (duration and end-point) based on timeline
-    // let timelineUpdated =
-    updateSegmentMetadata(playlist,
-                          currentMediaIndex,
-                          timelineUpdate);
+    updateSegmentMetadata(playlist, currentMediaIndex, timelineUpdate);
```

Comment:
This kind of change is just going to create merge conflicts. Specifically, when #626 is merged..